client/js/components/statement/splitStatement/SplitStatementView.vue

```diff
@@ -485,12 +483,47 @@ export default {
         this.prosemirror.view,
         rangeTrackerKey,
         editStateTrackerKey,
-        id,
-        { active: this.editingSegment.charEnd, fixed: this.editingSegment.charStart }
+        id
       )
       this.ignoreProsemirrorUpdates = false
     },
 
+    extractHtmlWithRangeMarks () {
+      /**
+       * Serialize the entire ProseMirror document to HTML, converting any `<span data-segment-id="...">…</span>` into a custom `<segment-mark>`
+       * element so that segment identifiers are preserved in the output.
+       *
+       * @param {EditorState} state
+       *  The current ProseMirror editor state.
+       * @returns {string}
+       *  Serialized HTML including `<segment-mark data-segment-id="...">…</segments-mark>`
+       */
```

Comment:
Thanks for commenting it.
Just: I think the Block should be above the method, not within.

Reply:
thx, done here https://github.com/demos-europe/demosplan-core/pull/4473/commits/ba741571752f908f15f4d656e8396d80c6a42ab2